crates/metastore/src/database.rs

```diff
@@ -707,9 +714,10 @@ impl State {
                         },
                     };
                     self.entries.insert(oid, CatalogEntry::Schema(ent))?;
-
+                
```

Comment:
Total nit (I'm not a primary maintainer on the repo, but am driving by): the spaces here might be accidental? I'm not sure that it matters, though.

Thanks for the fix :tada: !

Reply:
Thanks for the feedback, I appreciate it.
It's fixed now.